src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -24,8 +25,31 @@ export default class GroupComparisonMutationsTab extends React.Component<
     IGroupComparisonMutationsTabProps,
     {}
 > {
+    @observable public geneTab: string | undefined;
     constructor(props: IGroupComparisonMutationsTabProps) {
         super(props);
+        makeObservable(this);
+    }
+
+    @action.bound
+    protected handleGeneChange(id?: string) {
+        this.geneTab = id;
+        if (id) {
+            this.props.store.setSelectedMutationMapperGene(id);
+        } else {
+            this.props.store.clearSelectedMutationMapperGene();
+        }
+    }
+
+    @computed get tabs() {
+        const tabs: JSX.Element[] = [];
```

Comment:
this is nitpick but try to avoid declaring a variable and then filling it.  instead, just use your array methods.  this can instead be
this.props.store.genesWithMaxFrequency.map(g=><MSKTab key={g.hugoGeneSymbol} id={g.hugoGeneSymbol} linkText={g.hugoGeneSymbol} />));

Reply:
and you can just return that